src/rpc/net.cpp

```diff
@@ -84,8 +84,8 @@ UniValue getpeerinfo(const JSONRPCRequest& request)
             "                               autheticated via MNAUTH. In this case, this field contains the\n"
             "                               protx hash of the masternode\n"
             "    \"verified_pubkey_hash\":   h, (hex) Only present when the peer is a masternode and succesfully\n"
-            "                               autheticated via MNAUTH. In this case, this field contains the\n"
-            "                               hash of operator public key of the masternode\n"
+            "                               authenticated via MNAUTH. In this case, this field contains the\n"
```

Comment:
same typo above 🙈 but it's out of the scope for this PR 😄

Reply:
Don't you mean c09b83a? :D

Reply:
oh 🙈